var/spack/repos/builtin/packages/lua/package.py

```diff
@@ -37,6 +39,7 @@ class Lua(Package):
     # luarocks needs unzip for some packages (e.g. lua-luaposix)
     depends_on('unzip', type='run')
 
+    patch('add-lua-pc-file.patch', when="+pcfile")
```

Comment:
I think having the remote patch should work, issue is possibly a broken symlink (at least it is on my platform)

Reply:
@alalazo the updates in your rpm.patch.txt work for me as well! :tada: 

You reproduced my error perfectly (it looks like we are both on Ubuntu) - the lua.pc file was red when ti didn't work, and white when it did. I just added the applied patch here, and removed the patch file in the lua folder.